main.nf

```diff
@@ -87,17 +91,25 @@ include { run_clinical_release } from './modules/run_clinical_release'
 workflow BPC_PIPELINE {
    ch_cohort = Channel.value(params.cohort)
    ch_comment = Channel.value(params.comment)
-
-   run_quac_upload_report_error(ch_cohort)
-   run_quac_upload_report_warning(run_quac_upload_report_error.out, ch_cohort, params.production)
-   merge_and_uncode_rca_uploads(run_quac_upload_report_warning.out, ch_cohort, params.production)
-   // remove_patients_from_merged(merge_and_uncode_rca_uploads.out, ch_cohort, params.production)
-   update_data_table(merge_and_uncode_rca_uploads.out, ch_comment, params.production)
-   update_date_tracking_table(update_data_table.out, ch_cohort, ch_comment, params.production)
-   run_quac_table_report(update_date_tracking_table.out, ch_cohort, params.production)
-   run_quac_comparison_report(run_quac_table_report.out, ch_cohort, params.production)
-   create_masking_report(run_quac_comparison_report.out, ch_cohort, params.production)
-   update_case_count_table(create_masking_report.out, ch_comment, params.production)
+   
+   if (params.step == "update_potential_phi_fields_table") {
```

Comment:
I feel like testing only `update_potential_phi_fields_table` step can be done in its specific module. Or we can surface this step as a separate pipeline as https://tower.sagebionetworks.org/orgs/Sage-Bionetworks/workspaces/genie-bpc-project/launchpad/272225815007836

Reply:
Discussed this but you can set `params.step` to a specific step and run the workflow for just that step.